proposals/semi-auto-properties.md

```diff
@@ -54,7 +54,7 @@ The following changes are to be made to [classes.md](https://github.com/dotnet/c
 
 ```diff
 - If the auto-property has no set accessor, the backing field is considered `readonly` ([Readonly fields](classes.md#readonly-fields)). Just like a `readonly` field, a getter-only auto-property can also be assigned to in the body of a constructor of the enclosing class. Such an assignment assigns directly to the readonly backing field of the property.
-+ If the auto-property has semicolon-only get accessor without a set accessor, the backing field is considered `readonly` ([Readonly fields](classes.md#readonly-fields)). Just like a `readonly` field, a getter-only auto property can also be assigned to in the body of a constructor of the enclosing class. Such an assignment assigns directly to the backing field of the property.
++ If the auto-property has semicolon-only get accessor (without a set accessor or with an init-only accessor), the backing field is considered `readonly` ([Readonly fields](classes.md#readonly-fields)). Just like a `readonly` field, a getter-only auto property can also be assigned to in the body of a constructor of the enclosing class. Such an assignment assigns directly to the backing field of the property.
```

Comment:
`init-only accessor` or just `init accessor`

Reply:
```suggestion
+ If the auto-property has semicolon-only get accessor (without a set accessor or with an init accessor), the backing field is considered `readonly` ([Readonly fields](classes.md#readonly-fields)). Just like a `readonly` field, a getter-only auto property can also be assigned to in the body of a constructor of the enclosing class. Such an assignment assigns directly to the backing field of the property.
```

Reply:
Yup it's just `init accessor`.

Reply:
Not sure the added parens are right for this, since they aren't expanding into a more detailed version of the thing right before the parens.

Reply:
Maybe similar to this?

> If the auto-property has semicolon-only get accessor and has no set accessor (or has an init accessor),